{{cookiecutter.project_name}}/.github/workflows/run-tests.yml

```diff
@@ -77,9 +77,12 @@ jobs:
               }
             ]
           run-name: {{ cookiecutter.project_name}} Integration Test
+          {%- if cookiecutter.include_feature_store == "no" %}
```

Comment:
@arpitjasa-db It's fine as long as the condition is consistent across all places. Since other places are using `cookiecutter.include_feature_store == "yes"`, We should use `cookiecutter.include_feature_store != "yes"` here.

Reply:
Wait @mingyu89 I disagree. In other places in stacks like [here](https://src.dev.databricks.com/databricks/mlops-stack/-/blob/%7B%7Bcookiecutter.project_name%7D%7D/docs/mlops-setup.md?L46:21), we use `"no"`. I think it would be better to stay consistent with this convention.

Reply:
@arpitjasa-db  Ok. Then I'm Ok with fixing it in another PR. When we only check against one string, we are sure the condition is consistent for all places.

Reply:
It's not likely, but for example, if the value is "Yes" or "true", we treat it as "no feature store" across all places.(easier for user to figure out the outcome and the reason)

Reply:
Yeah I see your point, maybe we should do a better job instead of enforcing the schema described [here](https://src.dev.databricks.com/databricks/mlops-stack/-/blob/cookiecutter.json?L11:4).

Reply:
@arpitjasa-db  We can use this once 2.2.0 is available. https://cookiecutter.readthedocs.io/en/latest/advanced/boolean_variables.html